packages/chrome/src/elements/nav/NavItemIcon.ts

```diff
@@ -5,25 +5,19 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { Children } from 'react';
-import PropTypes from 'prop-types';
+import React, { Children, HTMLAttributes } from 'react';
 import ChromeStyles from '@zendeskgarden/css-chrome';
 
 const COMPONENT_ID = 'chrome.nav_item_icon';
 
-/** Applies styling directly to child component */
-const NavItemIcon = React.forwardRef(({ children }, ref) => {
-  return React.cloneElement(Children.only(children), {
+/**
+ * Applies styling directly to child component
+ **/
+export const NavItemIcon = React.forwardRef<any, HTMLAttributes<any>>(({ children }, ref) => {
+  return React.cloneElement(Children.only(children as any), {
```

Comment:
Is this intended to move to `styled` in the CSS-in-JS follow-on PR?

Reply:
I forgot to add the "moving to styled" comment to this component. I've added this comment to match `HeaderItemIcon`